### ISSUE TITLE
Area of interest input to advanced search

### ISSUE THREAD

**Issue by steveoh** [agrc/wri#22]
The WRI application has a search function that, when in the advanced mode, allows a user to draw a simple polygon shape or upload a shapefile.

- [x] Create a separate embeddable map application that can be placed in the `/wri/project/search.html` screen
- [ ] Coordinate with DTS to update the embed code
- [ ] Place a UI element on the screen to activate polygon drawing
- [ ] Place a UI element on the screen to accept a shapefile upload
- [ ] Allow the graphic to be cleared
- [ ] When the drawing is finished, set a hidden text field named `aoiGeometry` with the well known text for the geometry*

*We wrote our own conversion utility. I wonder if js4 or something open source is available. Otherwise our hack job could still function on 4x geometries.

### Prior implementation

![Image](https://github.com/user-attachments/assets/e92a0ad8-d279-4e7b-8e3d-16aa874e0a42)

#### References 

-  #8

**Comment by stdavis** [comment]
@steveoh Did you say that you had a test .zip file that the client has given you for testing shapefile uploads? Would you mind posting it here?

**Comment by steveoh** [comment]
yes. it was a problematic geometry for the feature creation upload. I think the vertices are close together. I'll drop it in the wri drive folder and share a link.
